Mergin/plugin.py

```diff
@@ -211,12 +211,19 @@ def project_status(self):
             if push_renamed:
                 push_msg += f"renamed: {push_renamed} \n"
 
+            files_to_replace = ""
+            for file in push_changes["updated"]:
+                if "diff" not in file and ".gpkg" in file['path']:
+                    files_to_replace += f"{file['path']}, "
+            files_to_replace.rstrip(", ")
```

Comment:
Better to build an array of strings and then do `", ".join(['hello', 'world'])` to join the items into a single string...